.github/workflows/ci_compatible.yml

```diff
@@ -81,31 +81,36 @@ jobs:
       matrix: ${{ fromJSON(needs.generator.outputs.matrix) }}
     steps:
       - name: Download 📥 summary
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
         with:
           name: compatibility-${{ github.sha }}
       - name: Download 📥 project
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
+        id: download-project
         with:
-          name: project-${{ github.sha }}-${{ strategy.job-index }}
+          pattern: project-${{ github.sha }}-*
 
       - run: ls -l
       - name: append result
         run: |
           import json
+          import os
           with open("projects.json") as fp:
             data = json.load(fp)
-          with open("meta.json") as fp:
-            data.append(json.load(fp))
+          for root, dirs, files in os.walk("${{ steps.download-project.outputs.download-path }}"):
```

Comment:
For w/e reason this output is not that useful https://github.com/actions/download-artifact?tab=readme-ov-file#outputs and the artifact does not unpack in the current directory. Added some logic to collect the meta files from the artifacts.